src/App.tsx

```diff
@@ -1,13 +1,65 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTodos } from './components/FilterTodos';
 
-const USER_ID = 0;
+import { FilterType } from './types/FilterType';
+import { ErrorType } from './types/ErrorType';
+
+const USER_ID = 10210;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState(FilterType.ALL);
+  const [errorType, setErrorType] = useState('');
+
+  const getTodosFromServer = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setErrorType(ErrorType.LOAD);
+    }
+  };
+
+  const filteredTodos = useMemo(() => {
+    return todos.filter(todo => {
+      switch (filterType) {
+        case FilterType.ALL:
+          return todo;
+
+        case FilterType.ACTIVE:
+          return !todo.completed;
+
+        case FilterType.COMPLETED:
+          return todo.completed;
+
+        default:
+          return todo;
+      }
+    });
+  }, [todos, filterType]);
+
+  const todosCount: [number, number] = useMemo(() => (
+    [
+      todos.filter(todo => !todo.completed).length,
+      todos.filter(todo => todo.completed).length,
+    ]
+  ), [todos]);
+
+  useEffect(() => {
+    getTodosFromServer();
+    const timer = setTimeout(() => {
```

Comment:
Setting a timer to clear the error message might not be the best approach, consider clearing the error message when the action is successful or there is a new user interaction.

Reply:
It's one of the tasks:
- automatically hide the notification after 3 seconds;